andela/ah-aquaman-frontend
# #164069194 see the time it takes to read an article

## What does this PR do?

This PR implements the feature of users being able to see the time it takes to read an article.

## Description of the task to be completed

- Display the time it takes to read on all articles
- Be able to view a single article including its read time

## How should this be manually tested?

- Clone this repository
- cd into the directory Ah-aquaman-frontend
- Check out the branch `ft-read-time-164069194`
- Run `npm install` to install all dependencies
- Run `npm start` to get the server running
- You will view the articles once your server starts at `localhost/`
- Then if you are logged in you are also able to view a specific article and the time it takes to read it

## What are the relevant pivotal tracker stories?

[#164069194](https://www.pivotaltracker.com/story/show/164069194)

## Screenshots

On the homepage, you are able to see some articles including their readtime

![image](https://user-images.githubusercontent.com/42435346/56577875-d145de80-65d4-11e9-94ca-db4e3782c5a0.png)
